### DISCUSSION TITLE
Notary Governance Issues | On record discussions

### DISCUSSION THREAD

**Discussion by dd45e640b42e6da7da96faee3996ef7c** [filecoin-project/notary-governance#827] [Ideas]
Good day!

we seek to get some unclear rules into writing that, after spending some time on the FIL+ program and around datacap applications (especially LDN), are not clear to us. We are not sure what clients and notaries (and to some extend SPs, as counter parties) can expect in regards to these issues.

**SPs collaborating with notaries**

originating in this comment of the FF Trust and Transparency Lead https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/928#issuecomment-1431365380 (and following ones). There is an almost identical statement on slack too https://app.slack.com/client/TEHTVS1L6/C0405HANNBT/thread/C0405HANNBT-1676468734.904469

We would like to discuss the issue on a notary call in early march. We especially seek the following outcome:

- clarify that it is ok for SPs to collaborate with notaries
- get the result in writing on one of the https://github.com/filecoin-project/notary-governance README pages

Having a clear, written down statement/rule on the topic will allow clients, SPs (and their collaborating notaries) to collaborate on FIL+ data storage facilitation in a way more efficient manner than currently possible. 

**Finding SPs through auctions**

For us it is not clear if trying to find SPs through deal auctioning platforms is a valid reason for a notary to oppose a clients datacap application. 

We would like to discuss the topic on a notary call in early march. We especially seek the following outcome:

- clarify that it is ok for notaries to sign an application that seeks to find SPs on deal auctioning platforms
- get the result in writing on one of the https://github.com/filecoin-project/notary-governance README pages

We use "deal auctioning platform" here as a placeholder for every kind of platform that seeks to connect clients with SPs utilizing an incentive mechanism.

Having a clear, written down statement/rule on the topic will allow clients, SPs (and thei collaborating notaries) to collaborate on FIL+ data storage facilitation in a way more efficient manner than currently possible. 

**SP geo location**

It is not clear to us if the  approval of  datacap requests is in any way bound to the geo location of the storing SPs. 

We would like to discuss the issue on a notary call in early march. We especially seek the following outcome:

- clarify that there is no requirement for clients to choose SPs through a certain geo location scheme
- get the result in writing on one of the https://github.com/filecoin-project/notary-governance README pages

Having a clear, written down statement/rule on the topic will allow clients, SPs (and their collaborating notaries) to collaborate on FIL+ data storage facilitation in a way more efficient manner than currently possible. 

**Online retrieval of FIL+'ed data**

There seems to be the expectation that FIL+ data stored with SPs has the obligation to be retrievable with filecoin native tooling, especially lotus' and boost's so called "online retrieval" mechanisms

We would like to discuss the issue on a notary call in early march. We especially seek the following outcome:

- clarify that "online retrieval" is not necessary for SPs to provide
- clarify that the client is responsible to provide a way to retrieve the data from SPs to potential retrievers
- get the result in writing on one of the https://github.com/filecoin-project/notary-governance README pages

Having a clear, written down statement/rule on the topic will allow clients, SPs (and their collaborating notaries) to collaborate on FIL+ data storage facilitation in a way more efficient manner than currently possible. 

**What we are not looking for**

Extended discussion on slack, extended discussions here in the github discussion thread. The topics have been discussed plenty. We purely seek discussion in an official, recorded forum (gov. call) and in the end the publication of the results via the "official" publication medium (github READMEs)

@gov-team please let us know when the notary governance team can find time to discuss the above topics on a governance call. We are, kinda, flexible but wish to get some preparation time (hence early march - maybe?) to make the case for our desired outcomes. 

We are happy to discuss potential dates, formats to get this over with in an efficient manner. 

contact: filecoin@factor8.io (if someone likes to discuss the issues off public record)

We hope to get some clarification so it will become possible to stream line datacap applications for the benefit of the client. The current climate in the FIL+ scene is not at all beneficial to the clients and we think a lot of the drama is rooted in the above, unclear issues. 

Thank you for your time!

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
we are not sure if the e-FIL+ governance is disconnected from the normal FIL+ governance, thats why we "outsource" this issue into a separated post:

**e-FIL+ random notary feature**

A notary that wasn't randomly assigned to a E-FIL+ application signed anyways: https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/928#issuecomment-1431302683. We seek resolution of this case. 

We would like to discuss the issue on a notary call in early march. We especially seek the following outcome:

- revoke the datacap allocation or retire the randomization feature
- define clear rules how to handle similar cases (if the randomization feature still exists after this)
- get the result in writing on one of the https://github.com/filecoin-project/notary-governance README pages

Having a clear, written down statement/rule on the topic will allow clients, SPs (and their collaborating notaries) to collaborate on FIL+ data storage facilitation in a way more efficient manner than currently possible.

**Reply by jamerduhgamer** [reply]
Good points made here. Should definitely discuss this in the next notary governance and e-FIL+ meetings.

**Comment by kevzak** [comment]
@f8-ptrk , I'll respond to some of [your points above](https://github.com/filecoin-project/notary-governance/discussions/827#discussioncomment-5003208) about the E-Fil+ notary assignment feature. 

E-Fil+ is a pilot program, under the same governance as Fil+, but with some leniency toward the testing of new tools and processes. 

We are currently focused on testing:
- Client trust tools (KYC/KYB)
- Notary accountability processes
- "Lead" SP ability to organize application details and onboard per the outlined plan

We do not have documentation posted on notary governance page yet because we are still in the pilot phase and nothing is finalized. However, every process currently being used was proposed in a [discussion topic](https://github.com/filecoin-project/notary-governance/discussions)  and was also reviewed on a notary governance call before starting, to allow for community feedback.

Regarding the application case that you seek resolution: 
- I have [replied directly in the application](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/928#issuecomment-1435401283)
   - I have asked the client to pause using the recently allocated Datacap until we can resolve the situation
 
Regarding the notary random assignment SLA: 
- All E-Fil+ Notaries were asked to review the [v1 SLA guidelines](https://github.com/filecoin-project/notary-governance/discussions/807) and confirm their agreement. 
- We did review [the pros and cons of this experiment](https://github.com/filecoin-project/notary-governance/discussions/810) including that the tooling today allows any notary in the multisig to sign and would require time to put in place.  

- Overall, the test has been a success thus far in my opinion. I haven't compiled all of the statistics yet, I'll do a more detailed analysis by the end of Q1, but in at least 10 applications thus far, it has increased engagement time by notaries (by asking them to leave comments within 72 hours) and has forced notaries to review applications and document their due diligence toward signing or declining to sign. 
- Yes, we did have one case recently that experienced a non-assigned notary signing. However per the notary, it was an accident and they did not intentionally break the rules.

- Based on all of this, I feel we should keep this experiment moving forward, the pros far outweigh the cons. Retiring this feature now in favor of an open multisig notary review would detract from all the accountability efforts we are testing. 
- Yes, it's clear in future iterations that there can be better tooling in place. For example: to only allow assigned notaries to sign or to proactively block any notaries involved with a client project from having the ability to sign. We can work with @panges2 to prioritize.
- And also, once the ability to revoke client DataCap is available, we can move to implement for a specific case like this one.

- I will proactively send a message to all notaries to remind them that for E-Fil+ applications to only sign if assigned. 
- In terms of a clear rule in place if this does happen again, we need to decide what measures to take. I'm opening to any feedback from the community.  I'm in favor of asking the client to pause usage until other notaries can review and decide. As is the current case with #928.

**Comment by sxxfuture-official** [comment]
It's really a good way to improve efficiency.
